### PR TITLE
Small README update for DataLoader2

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #543
* __->__ #541

Differential Revision: [D37394421](https://our.internmc.facebook.com/intern/diff/D37394421)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,10 +208,9 @@ the `DataLoader` as long as a `ShardingFiler` DataPipe exists in your pipeline.
 
 Q: What is the upcoming plan for DataLoader?
 
-A: There will be a new version of DataLoader in the next release. At the high level, the plan is that DataLoader V2 will
-only be responsible for multiprocessing, distributed, and similar functionalities, not data processing logic. All data
-processing features, such as the shuffling and batching, will be moved out of DataLoader to DataPipe. At the same time,
-the current/old version of DataLoader should still be available and you can use DataPipes with that as well.
+A: DataLoader2 is in the prototype phase and more features are actively being developed. Please see the README file in
```
Comment (ejguan) [inline README.md:211]: nit: `Dataloader2`

File: README.md [hunk 2]
```diff
@@ -208,10 +208,9 @@ the `DataLoader` as long as a `ShardingFiler` DataPipe exists in your pipeline.
...
+`torchdata/dataloader2`. If you would like to experiment with it (or other prototype features), we encourage you to
```
Comment (ejguan) [inline README.md:212]: You can link README to https://github.com/pytorch/data/blob/main/torchdata/dataloader2/README.md

**Review by ejguan — APPROVED** [review]
LGTM. Thank you

File: README.md (hunk 1 above)
Comment (ejguan) [reply README.md:211]: Wrap DataLoader2 with "`"

**Comment by NivekT** [comment]
I also going to fix the link mentioned by one of your issue. Let me push another commit soon.

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D37394421).

**Pull request closed without merging**